src/main/resources/licenses/licenses-transform.xsl

```diff
@@ -227,7 +227,7 @@
           <xsl:with-param name="url" select="$cddl_url"/>
         </xsl:call-template>
       </xsl:when>
-      <xsl:when test="contains(url/text(), 'glassfish.java.net/public/CDDL+GPL') or contains(url/text(), 'glassfish.dev.java.net/nonav/public/CDDL+GPL') or contains(url/text(), 'glassfish.dev.java.net/public/CDDL+GPL')">
+      <xsl:when test="contains(url/text(), 'glassfish.java.net/public/CDDL+GPL') or contains(url/text(), 'glassfish.dev.java.net/nonav/public/CDDL+GPL') or contains(url/text(), 'glassfish.dev.java.net/public/CDDL+GPL') or contains(url/text(), 'https://glassfish.dev.java.net/public/CDDLv1.0.html')">
```

Comment:
This doesn't sounds right. `$cddl_name` is `Common Development and Distribution License (CDDL) and GNU Public License v.2 w/Classpath Exception`, while the license in question seems to be pure CDDL.

Reply:
Sorry, I looked at the comment above that code only which says CDDL...The good news this particular condition did not trigger for -redhat anyway (there was no CCDL related test warning as it was for -community, and then I did check license.xml before even modifying the stylesheet and a pure CDDL name did match what I can see now a cddl_10_name var). I'll create a temp community branch for one of the boosters and test